### DISCUSSION TITLE
sFTP Go - Auto restart

### DISCUSSION THREAD

**Discussion by sharansaikumar** [drakkan/sftpgo#1153] [Q&A]
Hello,

I am having an issue where the sFTPGo service is auto-restarting on redhat linux virtual machine. We tried to change the default port it ran and I don't know if that impacted running the service. 
Attached screenshot. 
Please suggest on how to fix the auto-restart issue? 
![sftpgo issue](https://user-images.githubusercontent.com/55262198/213004521-46714a61-3b9e-40d6-b378-5e210a39d837.JPG)

**Comment by drakkan** [comment]
check your SFTPGo logs, the answer is there
